tests/data/sample_specification.yaml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
This test specification has been beefed up to accommodate all new test cases, including dialogue part which was previously missing.